doc/source/basics.rst

```diff
@@ -624,6 +624,66 @@ We can also pass infinite values to define the bins:
 Function application
 --------------------
 
+There are three main cases for function, depending on what the function
```

Comment:
-> To apply your own or another library's functions to pandas objects,
you should be aware of the three methods below.